lib/checknullpointer.cpp

```diff
@@ -322,6 +322,8 @@ static bool isNullablePointer(const Token* tok, const Settings* settings)
 {
     if (!tok)
         return false;
+    if (Token::simpleMatch(tok, "new"))
```

Comment:
Could this lead to false results for C code with new being variable/function/etc. ?

Reply:
It could lead to FN, let me update that.